src/orders/utils/data.ts

```diff
@@ -320,6 +336,20 @@ export const getFulfillmentFormsetQuantity = (
 ) => formsetData.find(getById(line.id))?.value?.[0]?.quantity;
 
 export const getWarehouseStock = (
-  stocks: OrderFulfillLineFragment["variant"]["stocks"],
+  stocks: OrderLineFragment["variant"]["stocks"],
   warehouseId: string
 ) => stocks?.find(stock => stock.warehouse.id === warehouseId);
+
+export const isLineAvailableInWarehouse = <T extends OrderFulfillLineStockData>(
+  line: T,
+  warehouse: Warehouse
+) => {
+  if (!line?.variant?.stocks) {
+    return false;
+  }
+  const stock = getWarehouseStock(line.variant.stocks, warehouse.id);
+  if (stock) {
+    return line.quantityToFulfill <= getOrderLineAvailableQuantity(line, stock);
+  }
+  return false;
```

Comment:
```suggestion
  return stock ?? line.quantityToFulfill <= getOrderLineAvailableQuantity(line, stock);
```
Correct me if I'm wrong but I think it would work the same, no?

Reply:
No. Function starts with `is` and returns a boolean value, not a stock object